Dockerfile

```diff
@@ -1,9 +1,4 @@
-FROM registry.tld/cyberark/ubuntu-ruby-fips:latest
-
-RUN export DEBIAN_FRONTEND=noninteractive && \
-  apt-get update -y && \
-  apt-get dist-upgrade -y && \
-  apt-get install -y build-essential curl git tzdata
```

Comment:
Does this new base image include all of the tools that are being installed here? (`build-essential curl git tzdata`)

Reply:
Yes I think so. I've checked image and below is the result.
<img width="1009" alt="image" src="https://github.com/conjurinc/rack-heartbeat/assets/131350103/0d984068-d5da-4379-8d23-b5713283e3eb">

Reply:
Ach, it may not have all of them. that PR was initially made for newer Ubuntu images where all of these tools is installed.
I'll revert that deletion. Thanks for that comment!